src/index.html

```diff
@@ -4,9 +4,371 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,700&display=swap"
+      rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        src="./images/logo.png"
+        class="header__logo"
+        alt="logo"
+      >
+      <nav class="nav">
+        <ul class="nav__menu">
+          <li class="nav__menu-li">
+            <a class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="nav__menu-li">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__menu-li">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__menu-li">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__menu-li">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__menu-li">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+          <li class="nav__menu-li">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+          <li class="nav__menu-li">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container main__container">
+        <section class="card container__card" data-qa="card">
+          <div class="card__image-container">
+            <img
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            >
+          </div>
+          <div class="card__title-container">
+            <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          </div>
+
+          <div class="card__code-container">
+            <p class="card__code">Product code: 195434</p>
+          </div>
```

Comment:
Why do you wrap into `<div>`? No needs
Check all over the page
```suggestion
            <p class="card__code">Product code: 195434</p>
```